src/com/oltpbenchmark/benchmarks/wikipedia/procedures/UpdatePage.java

```diff
@@ -247,13 +247,13 @@ public void run(Connection conn, int textId, int pageId,
 			conn.commit();
 
 			ps = this.getPreparedStatement(conn, updateWatchList);
-			param = 1;
-			ps.setString(param++, timestamp);
-			ps.setString(param++, pageTitle);
-			ps.setInt(param++, pageNamespace);
 			for (Integer otherUserId : wlUser) {
-				ps.setInt(param, otherUserId.intValue());
-				ps.addBatch();
+			    param = 1;
```

Comment:
What is this change?

Reply:
In the previous code, each time it creates a prepared statement, it rebinds the `userId` field and does not change other fields. 
For Cubrid, it complains that in the generated statements, not all parameters are binded. From the log record in Cubrid Broker, in the query that server received, only `userId` is binded and other params are not. So we rebind all the params here.